components/formats-api/src/loci/formats/IFormatReader.java

```diff
@@ -326,18 +338,44 @@ Object openPlane(int no, int x, int y, int w, int h)
    */
   boolean isOriginalMetadataPopulated();
 
-  /** Specifies whether or not to force grouping in multi-file formats. */
+  /**
+   * Specifies whether or not to force grouping in multi-file formats.
+   *
+   * @see #fileGroupOption(String)
+   * @see #isGroupFiles()
+   */
   void setGroupFiles(boolean group);
 
-  /** Returns true if we should group files in multi-file formats.*/
+  /**
+   * Returns true if we should group files in multi-file formats.
+   *
+   * @see #setGroupFiles(boolean)
+   * @see #fileGroupOption(String)
```

Comment:
`Tag @see: can't find fileGroupOptions(String) in loci.formats.IFormatReader` - see https://ci.openmicroscopy.org/job/BIOFORMATS-DEV-merge-build/1119/warnings42Result/

Reply:
Should already be fixed by 1301ad0